Detectors/TRD/calibration/src/PadCalibCCDBBuilder.cxx

```diff
@@ -0,0 +1,551 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file PadCalibCCDBBuilder.cxx
+/// \brief Krypton calibration - class to smoothen and inter/extrapolate gain in chambers and calculate normalized ADC gain per chamber
+/// \author Jana Crkovska
+
+#include "TRDCalibration/PadCalibCCDBBuilder.h"
+#include <iostream>
+#include <string>
+#include <map>
+#include <memory>
+#include "TFile.h"
+#include "TH2.h"
+#include "TMath.h"
+#include "TString.h"
+#include "TTree.h"
+
+using namespace o2::trd::constants;
+
+namespace o2::trd
+{
+
+void PadCalibCCDBBuilder::checkIfIsolatedHotPadCandidate(TH2F* hDet, std::vector<int> coordinates, float upperLimit, int areaContainedWithin)
+{
+  if ((int)coordinates.size() != 4) {
+    std::cerr << "Invalid size of the coordinates vector!" << std::endl;
+    return;
+  }
```

Comment:
Using an array or a struct would remove the need to test the size of the coordinates vector.
The struct would also clarify the meaning of the 4 elements of the vector. The code looks like there are two pairs of pad row/column coordinates, but I'm not sure what the difference is.